src/components/TodosList.jsx

```diff
@@ -0,0 +1,15 @@
+import TodoItem from './TodoItem';
+
+const TodoList = ({ todosProps, handleChange, delTodo }) =>
+
+// const {todosProps} = props;
```

Comment:
- please, don't forget to remove any commented code. as it might not be professional.